internal/common/agent_util_unix.go

```diff
@@ -66,6 +66,33 @@ func TouchFile(filePathAbsolute string) error {
 	return nil
 }
 
+func printOutputAndError(function string, stdout []byte, err error) {
```

Comment:
My main goal was to avoid duplicating the code below:
```
	if err == nil {
		return
	}
	stderr := ""
	ee, ok :=  err.(*exec.ExitError)
	if ok {
		stderr = string(ee.Stderr)
```
in both `InstallAgent` and `UninstallAgent`.
Basically any function that calls `exec.Command()` followed by `c.Output()` could/should be logging both the stdout AND stderr.

I will rename the variable.